my_list.rb

```diff
@@ -0,0 +1,24 @@
+require_relative 'MyEnumerable'
+
```

Comment:
I got this error while trying to verify your solution. 👀

![image](https://github.com/mumo-esther/Rubybs/assets/78251927/08adc6f3-7a4c-4a1e-9981-628eabbd0690)

You have used `require_relative` to import `MyEnumerable` wrongly 🙄, `require_relative` should be used for referring to files within your directory." 🤓

i.e 👍 
```
require_relative 'my_enumerable
```

Please kindly fix this 🛠